tests/playwright/pages/editor-page.js

```diff
@@ -242,4 +277,57 @@ module.exports = class EditorPage extends BasePage {
 	async removeFocus() {
 		await this.getPreviewFrame().locator( '#elementor-add-new-section' ).click( { button: 'right' } );
 	}
+
+	/**
+	 * Hide all editor elements from the screenshots.
+	 *
+	 * @return {Promise<void>}
+	 */
+	async hideEditorElements() {
+		const css = '<style>.elementor-element-overlay,.elementor-empty-view,.elementor-widget-empty,.e-view{opacity: 0;}.elementor-widget,.elementor-widget:hover{box-shadow:none!important;}.elementor-add-section-inner {border: none !important;background-color: #cccccc !important;}</style>',
+			widgetContainer = await this.addElement( { elType: 'container' }, 'document' ),
+			htmlWidget = await this.addWidget( 'html', widgetContainer );
+
+		await this.getPreviewFrame().waitForSelector( '.elementor-widget-html' );
+		await this.selectElement( htmlWidget );
+		await this.page.locator( '.elementor-control-type-code textarea' ).fill( css );
+	}
+
+	/**
+	 * Hide controls from the video widgets.
+	 *
+	 * @return {Promise<void>}
+	 */
+	async hideVideoControls() {
+		await this.getPreviewFrame().waitForSelector( '.elementor-video' );
+
+		const videoFrame = this.getPreviewFrame().frameLocator( '.elementor-video' ),
+			videoButton = videoFrame.locator( 'button.ytp-large-play-button.ytp-button.ytp-large-play-button-red-bg' ),
+			videoGradient = videoFrame.locator( '.ytp-gradient-top' ),
+			videoTitle = videoFrame.locator( '.ytp-show-cards-title' ),
+			videoBottom = videoFrame.locator( '.ytp-impression-link' );
+
+		await videoButton.evaluate( ( element ) => element.style.opacity = 0 );
+		await videoGradient.evaluate( ( element ) => element.style.opacity = 0 );
+		await videoTitle.evaluate( ( element ) => element.style.opacity = 0 );
+		await videoBottom.evaluate( ( element ) => element.style.opacity = 0 );
+	}
```

Comment:
@hein-obox is there a reason for `opacity: 0;` instead of `display:none;` for example? And just a question, why do we need to use YouTube videos in our screenshots if they cause so much problems?

Reply:
@louiswol94 

- `opacity` vs `display`: my reasoning was that the `display` type could have been defined in the third party styling already. I am not 100% sure about how our `js` codes related to existing styling in the `iframe`, but I don't think it makes much of a difference.
- youtube videos: there is some styling which is specific for the `video` and the `maps` widgets, and therefore I believe it is good to test it. Hopefully this PR will make the tests more reliable. If not, we can discuss it again.